data_steward/validation/main.py

```diff
@@ -340,14 +349,21 @@ def get_heel_result(hpo_id, app_id=None, dataset_id=None):
     return result
 
 
-def get_heel_error_header(list_of_dicts):
-    fake_header_list = ['Record Count', 'Heel Error', 'Analysis ID', 'Rule ID']
-    if not list_of_dicts:
-        return fake_header_list
+def _convert_query_result_to_row_list(list_of_dicts):
+    row_list = list()
+    if list_of_dicts is []:
```

Comment:
This could be changed according to [PEP 8](https://www.python.org/dev/peps/pep-0008/):
```md
For sequences, (strings, lists, tuples), use the fact that empty sequences are false.

Yes: if not seq:
     if seq:

No:  if len(seq):
     if not len(seq):
```

Reply:
Also interesting to note that
```python
l = []
print(l == []) # True
print(l is []) # False
```

Reply:
Also this not necessary, since an empty list of iterables can still be iterated through, but the inner loop won't be traversed. So the following code should suffice:
```python
def _convert_query_result_to_row_list(list_of_dicts):
    row_list = list()
    for dict_item in list_of_dicts:
        dict_values = tuple(dict_item.values())
        row_list.append(dict_values)
    return row_list
```